recipes/bison/all/conanfile.py

```diff
@@ -150,6 +150,6 @@ def package_info(self):
         self.output.info('Setting the BISON_PKGDATADIR environment variable: {}'.format(pkgdir))
         self.env_info.BISON_PKGDATADIR = pkgdir
 
-        yacc_bin = os.path.join(self.package_folder, "bin", "yacc").replace("\\", "/")
-        self.output.info("Setting YACC environment variable: {}".format(yacc_bin))
-        self.env_info.YACC = yacc_bin
+        # yacc_bin = os.path.join(self.package_folder, "bin", "yacc").replace("\\", "/")
+        # self.output.info("Setting YACC environment variable: {}".format(yacc_bin))
+        # self.env_info.YACC = yacc_bin
```

Comment:
With this set, I get stuff in my build log (trying to build ruby):

```
  generating parse.c
  'C:\Users\julien\.conan\data\bison\3.7.1\_\_\package\bb0cfedda536453d378977a7c3afe22a6701a2bc\bin\yacc' is not recognized as an internal or external command,
  operable program or batch file.
NMAKE : warning U4010: 'parse.c' : build failed; /K specified, continuing ...
```

Commenting it out makes it work

Reply:
`yacc` is a bash utility, if you try running it from cmd.exe on windows you get a dialog box saying it's not compatible with 64 bit systems.